app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const app = express();
+
+const { getApi } = require("./controllers/api.controller");
+
+app.use(express.json());
```

Comment:
With regards to TDD, what is this doing and do we need it to pass our test at this point?